src/ec.rs

```diff
@@ -791,7 +787,7 @@ where
   fn tell(&mut self) -> u32 {
     // The 10 here counteracts the offset of -9 baked into cnt, and adds 1 extra
     // bit, which we reserve for terminating the stream.
-    (((self.stream_bytes() * 8) as i32) + (self.cnt as i32) + 10) as u32
+    (((self.stream_bits()) as i32) + (self.cnt as i32) + 10) as u32
```

Comment:
I think this technically should have an AWCY run, as we actually change the result of tell() for WriterCounter.

Reply:
https://beta.arewecompressedyet.com/?job=master-90fdd3dec515600b7a9ab67ed071e5a88f804cee&job=maj-optim-a6%402023-01-18T12%3A50%3A49.710Z
I'll attach the results when it completes.

Reply:
I have been running each of these commits locally to ensure the output videos are identical. Looks like AWCY agrees, doesn't hurt to have it run for the extra thoroughness.